### ISSUE TITLE
Right side tile border doesn't match left side

### ISSUE THREAD

**Issue by croossin** [rpocklin/angular-timeline#34]
It seems that the z-index or something may be off.  As you can see in the screenshot, the left-box and arrow pointing towards the timeline act as "one", while the right-side still has a border covering part of the arrow.
<img width="160" alt="screen shot 2016-06-16 at 12 31 57 pm" src="https://cloud.githubusercontent.com/assets/7241069/16130530/7a90ac1e-33be-11e6-8fd7-eab7b088ce90.png">

**Comment by rpocklin** [comment]
Yep you're right.  Looks like a CSS artifact, not sure when it came in.  Should be easy to fix i'll get to it soon thanks for finding it.

**Comment by croossin** [comment]
Sweet.  I'll take a look in a bit too and may push a PR. Cheers
